http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/CommonTestData.scala

```diff
@@ -198,6 +198,12 @@ trait CommonTestData{ this: ScalaFutures =>
   protected def modifyInstanceKey(instanceKey: InstanceKey): InstanceKey = {
     instanceKey.copy(name = InstanceName(instanceKey.name.value + "_2"))
   }
+
+  protected class LeoMockGoogleIamDAO extends MockGoogleIamDAO {
```

Comment:
The wb-libs `MockGoogleIamDAO` didn't have quite the right behavior for `findServiceAccount`, so I extended it here.